src/MeoAssistant/Utils/AsstUtils.hpp

```diff
@@ -284,7 +300,7 @@ namespace asst::utils
         time_t nowtime = tv.tv_sec;
         struct tm* tm_info = localtime(&nowtime);
         strftime(buff, sizeof(buff), "%Y-%m-%d %H:%M:%S", tm_info);
-        sprintf(buff, "%s.%03ld", buff, tv.tv_usec / 1000);
+        sprintf(buff, "%s.%03d", buff, tv.tv_usec / 1000);
```

Comment:
要不直接
```suggestion
        sprintf(buff, "%s.%03ld", buff, static_cast<long int>(tv.tv_usec / 1000));
```
不然正常平台上会吐一大堆警告
>  warning: format specifies type 'int' but the argument has type 'long' [-Wformat]